samples/react-spupsproperty-sync/gulpfile.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// const build = require('@microsoft/sp-build-web');
+
+// build.addSuppression(`Warning - [sass] The local CSS class 'ms-Grid' is not camelCase and will not be type-safe.`);
```

Comment:
I would recommend re-adding this suppression and one more:
build.addSuppression('Warning - [sass] The local CSS class 'ms-Grid' is not camelCase and will not be type-safe.');

so that people who are inexperienced with SPFx don't get scared with warnings.

Reply:
@hugoabernier Sure will do that.